chainerio/_typing.py

```diff
@@ -0,0 +1,24 @@
+import sys
+import typing
+
+
+g = globals()
```

Comment:
Can you import only `optional` and `union` from `_typing` and others from `typing`, to make things clear